lazysizes-umd.js

```diff
@@ -70,19 +70,36 @@
 		});
 	};
 
-	var triggerEvent = function(elem, name, detail, noBubbles, noCancelable){
-		var event = document.createEvent('Event');
+	var buildEvent = function(name, detail, noBubbles, noCancelable) {
+		// https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill
+		var CustomEvent = window.CustomEvent;
+		if (typeof CustomEvent !== 'function') {
+			CustomEvent = function(event, params) {
```

Comment:
I would move the polyfilling part outside of the function so that it is not polyfilled everytime you generate an event. As soon as you have done this `buildEvent` is so small that it can be re-added to `triggerEvent`.